src/test/java/org/cactoos/scalar/AndInThreadsTest.java

```diff
@@ -163,4 +169,170 @@ public void testFunc() throws Exception {
         );
     }
 
+    @Test
+    public void testProcIterator() throws Exception {
```

Comment:
@driver733 can you change the test names from ``test...`` to ``worksWith...`` -- as in "class AndInThreads works with given Proc or somethnig else"